Gruntfile.js

```diff
@@ -91,7 +93,7 @@ module.exports = function (grunt) {
               replacement: "//fonts.googleapis.com",
             },
             {
-              match: /\.\.\/images\/([a-zA-Z0-9\-_@]*\.(png|jpg|svg))/g,
+              match: /\.\.\/images\/([\w@-]*\.(png|jpg|svg))/g,
```

Comment:
Sorry, I added a linter that simplifies regexes ad part of the unicorn/recommended pack. I think the idea of it is nice but I don't necessarily find this any easier to read, what do you think?

Reply:
I don't think it's easier to read but if it still works that's OK. I am just mildly nervous because in the past when I was experimenting with this file the regexes didn't seem to behave as I expected them to.

Reply:
@akegan that's a really good point, that regex parsers all have particular rules that the linting rule might not account for. That's enough of a reason for me to turn this off!

I'll change the linting rule to be a warning (or a "suggestion" if something less than a warning is possible) so we can learn from it if we write our own regexes but it doesn't automatically change our code.

Reply:
@akegan removed and reverted this change